packages/experience/src/pages/SignIn/IdentifierSignInForm/index.tsx

```diff
@@ -117,6 +122,8 @@ const IdentifierSignInForm = ({ className, autoFocus, signInMethods }: Props) =>
             isDanger={!!errors.identifier || !!errorMessage}
             errorMessage={errors.identifier?.message}
             enabledTypes={enabledSignInMethods}
+            defaultType={identifierInputValue?.type}
+            defaultValue={identifierInputValue?.value}
```

Comment:
Not sure how would this affect the input field's behavior better carefully test it.

Reply:
Tested.